src/asm/opt.c

```diff
@@ -71,16 +78,31 @@ void opt_Parse(char *s)
 			error("Must specify exactly 4 characters for option 'g'\n");
 		break;
 
+	case 'f':
+		if (strlen(&s[1]) <= 2) {
+			int result;
+			unsigned int precision;
+
+			result = sscanf(&s[1], "%u", &precision);
+			if ((result != EOF && result != 1) || precision < 1 || precision > 31)
+				error("Invalid argument for option 'f'\n");
+			else
+				opt_F(precision);
+		} else {
+			error("Invalid argument for option 'f'\n");
+		}
+		break;
+
 	case 'p':
 		if (strlen(&s[1]) <= 2) {
 			int result;
-			unsigned int fillchar;
+			unsigned int fill;
```

Comment:
`fillByte`, I guess.

Reply:
That would conflict with the `extern uint8_t fillByte` from `section.h`.

Reply:
Since it's destined to become `fillByte`, what about `cliFillByte`?

Reply:
Kind of a long name for a variable local to this small scope, but okay.